gradle-plugin/build.gradle

```diff
@@ -20,6 +21,8 @@ buildscript {
 repositories {
   google()
   mavenCentral()
+  // Required for publishing, Dokka needs org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.2
```

Comment:
Is there a public issue somewhere to get kotlinx-html-jvm published on maven central?

Reply:
There actually is: https://github.com/Kotlin/kotlinx.html/issues/173